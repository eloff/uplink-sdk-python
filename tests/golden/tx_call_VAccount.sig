r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 9303843980400483712830279429123129993749070216529550821723079731427395029331
