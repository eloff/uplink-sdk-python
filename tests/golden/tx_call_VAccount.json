{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                "args": [
                    {
                        "contents": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                        "tag": "VAccount"
                    }
                ], 
                "method": "get"
            }, 
            "tag": "Call"
        }, 
        "tag": "TxContract"
    }, 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AEw5MzAzODQzOTgwNDAwNDgzNzEyODMwMjc5NDI5MTIzMTI5OTkzNzQ5MDcwMjE2NTI5NTUwODIxNzIzMDc5NzMxNDI3Mzk1MDI5MzMx"
}